irods/parallel.py

```diff
@@ -380,12 +358,13 @@ def bytes_range_for_thread( i, num_threads, total_bytes,  chunk ):
     for byte_range in ranges:
         if Io is None:
             Io = session.data_objects.open( Data_object.path, Operation.data_object_mode(initial_open = False),
-                                            create = False, finalize_on_close = False,
+                                            create = False, finalize_on_close = False, allow_redirect = False,
                                             **{ kw.NUM_THREADS_KW: str(num_threads),
                                                 kw.DATA_SIZE_KW: str(total_size),
                                                 kw.RESC_HIER_STR_KW: hier_str,
                                                 kw.REPLICA_TOKEN_KW: replica_token })
         mgr.add_io( Io )
+        logger.info('target_host = %s', Io.raw.session.pool.account.host)
```

Comment:
may not want this at 'info'

Reply:
Right, this is debug only. May or may not be included in final version, anyway....